src/events/messageCreate.ts

```diff
@@ -204,6 +209,15 @@ export default async (client: CustomClient, message: Message): Promise<void> =>
                   "You can find more information about it [here](https://discord.com/channels/474583323340046337/474583324396879884/1067218193912963172)"
               ),
           ],
+          components: [
+            new ActionRowBuilder<ButtonBuilder>()
+              .addComponents(
+                new ButtonBuilder()
+                  .setCustomId(triggerId)
```

Comment:
`triggerId` should probably be related to which trigger it is, maybe something like a `trigger-${triggerName}`